lib/rules/sort-character-class-elements.ts

```diff
@@ -296,17 +296,17 @@ function escapeRaw(node: CharacterClassElement, target: CharacterClassElement) {
         ) {
             raw = `\\${raw}`
         }
-    } else if (raw.startsWith("^")) {
+    } else if (raw[0] === "^") {
         const parent = target.parent as CharacterClass
         const elements: (
             | UnicodeSetsCharacterClassElement
             | ClassRangesCharacterClassElement
         )[] = parent.elements
-        if (elements.indexOf(target) === 0) {
+        if (elements[0] === target) {
```

Comment:
This change is fine, but the others are unacceptable because they violate this project's policies.
Could you please revert the other changes?

Reply:
Done.

Reply:
I didn't explain it well.

Could you revert the change to `startsWith()`, leaving `elements[0] === target`?

Also, please check the CI error.

Reply:
Sorry, I misunderstood what you meant just now. I have corrected it.